### ISSUE TITLE
Put ad hoc reports in an ad_hoc/ subdirectory

### ISSUE THREAD

**Issue by jacobmpeters** [Analyticsphere/ccc_module_metrics_gcp_pipeline#20]
Hi Kelsey!

This is not urgent, but could you please move the ad hoc scripts to a separate subdirectory? This will help me to ignore those files when I'm debugging the pipeline issues.

Thanks!

**Comment by KELSEYDOWLING7** [comment]
Absolutely! Is it easier for me to create a sub folder in my local repo before pushing it? That way that subfolder is created in Github?

Or because they're already in there should we make subfolders in Github and I can do a pull?

**Comment by jacobmpeters** [comment]
Thanks! I think that making the folder locally and then pushing it will work fine. And no need to push it until your next update.
